Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 if (utils.scm_checkout()) return
 
 matrix_python = ['3.6']
-matrix_numpy = ['1.15.2']
+matrix_numpy = ['1.14']
```

Comment:
If you do want to merge this PR, make one more change after the GIthub issues blow over.  Change this back to `1.15.2`.  The tests should pass, and we can all have a nice weekend.